main-app/tests/audio-tests/audioServiceTests.js

```diff
@@ -1,18 +1,39 @@
 
-(function(){
+(function() {
     'use strict';
-    describe('AudioService Tests', function(){
-        var $document;
-        var $timeout;
+    describe('AudioService Tests', function() {
+        var sandbox;
+        var audioService;
         var audioSprite;
+        var audioSpriteMock;
```

Comment:
theres no need to use var to declare all of these, just state it the once and seperate the others with a comma. eg.
var sandbox,
      audioService;